common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -290,3 +290,9 @@ def set_wrong_inputs_to_fields(self):
                 "Course Announcement Date": '"string"',
             }
         )
+
+    def test_hidden_fields_not_displayed(self):
```

Comment:
@cahrens -- added the new test approach -- let me know what you think -- looks like Jenkins approves.